clients/cloud/ccloud/README.md

```diff
@@ -0,0 +1,134 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud).
+
+# Prerequisites
+
+* Local install of [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.ccloud-stacks) v1.13.0 or later
+* You must have access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
+
+  * The first 20 users to sign up for [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) and use promo code ``C50INTEG`` will receive an additional $50 free usage ([details](https://www.confluent.io/confluent-cloud-promo-disclaimer/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud)).
+
+# Example 1: Hello World!
+
+In this example, the producer writes Kafka data to a topic in your Kafka cluster. 
+Each record has a key representing a username (e.g. `alice`) and a value of a count, formatted as json (e.g. `{"count": 0}`).
+The consumer reads the same topic.
+
+1. Create the topic in Confluent Cloud
+
+```bash
+$ ccloud kafka topic create test1
+```
+
+2. Run the [Confluent Cloud CLI producer](https://docs.confluent.io/current/cloud/cli/command-reference/kafka/topic/ccloud_kafka_topic_produce.html#ccloud-kafka-topic-produce?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud), writing messages to topic `test1`, passing in additional arguments:
+
+* `--parse-key --delimiter ,`: pass key and value, separated by a comma
+
+```bash
+$ ccloud kafka topic produce test1 --parse-key --delimiter ,
+```
+
+Type a few messages, using a `,` as the separator between the message key and value:
+
+```bash
+alice,{"count":0}
+alice,{"count":1}
+alice,{"count":2}
+```
+
+When you are done, press `<ctrl>-d`.
```

Comment:
Sometimes for exiting, we instruct `<ctrl>-c` and other times we use `<ctrl>-d`.  IMO we stick with `<ctrl>-c`